internal/pkg/keybindings/list_update.go

```diff
@@ -130,88 +103,27 @@ func (h *ListUpdateHandler) Invoke() error {
 
 	case interfaces.ResponseTerraform:
 		fileExtension = ".tf"
-		formattedContent = content // TODO: add Terraform formatter
+		formattedContent = content // TODO: add Terraform fo``rmatter
```

Comment:
nit: looks like some extra chars snuck in.

```suggestion
		formattedContent = content // TODO: add Terraform formatter
```